code/game/objects/items/weaponry.dm

```diff
@@ -242,7 +242,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 		return INITIALIZE_HINT_QDEL
 
 /obj/item/claymore/highlander/robot/process()
```

Comment:
do you uh, know why this is on process?

Reply:
No, but if I had to deal with every bit of awful code I stumbled upon while working on this, things would have gone out of scope pretty fast.